packages/ui-extensions/src/surfaces/customer-account/components/CustomerAccountAction/CustomerAccountAction.doc.ts

```diff
@@ -10,21 +10,20 @@ const data: ReferenceEntityTemplateSchema = {
   type: '',
   definitions: [
     {
-      title: 'CustomerAccountActionProps',
+      title: 'Properties',
       description: '',
-      type: 'CustomerAccountActionProps',
+      type: 'CustomerAccountActionPropsDocs',
     },
     {
-      title: 'Button slot primary-action',
-      description:
-        'Supported props for Buttons used inside CustomerAccountAction `primary-action` slot.<br><br>`children` only support text.',
-      type: 'Docs_CustomerAccountAction_Button_PrimaryAction',
+      title: 'Slots',
```

Comment:
Checkout ([example](https://shopify.dev/docs/api/checkout-ui-extensions/2025-10-rc/polaris-web-components/overlay/modal#slots)) and Admin ([example](https://shopify.dev/docs/api/admin-extensions/2025-10-rc/polaris-web-components/feedback/banner#slots-propertydetail-secondaryactions)) provide a "Slots" section to list out all the slots on a component. I think that's a good idea to see them all together. I left the other prop definitions when they were different from the regular component.